bin/database.py

```diff
@@ -93,6 +93,7 @@ def process_cursor(cursor, closure):
     # pool.close()
     # pool.join()
 
+    logging.info('Proccessing {} items in cursor ...'.format(cursor.count()))
```

Comment:
@kofalt I was planning on adding "`n` of `count` items processed..." somewhere around here. Maybe divide it into 10 or 20 groups. Just didn't want to make it part of this PR.